libsolidity/codegen/CompilerUtils.cpp

```diff
@@ -1552,10 +1552,13 @@ void CompilerUtils::storeStringData(bytesConstRef _data)
 unsigned CompilerUtils::loadFromMemoryHelper(Type const& _type, bool _fromCalldata, bool _padToWords)
 {
 	solAssert(_type.isValueType(), "");
+	Type const* type = &_type;
+	if (auto const* userDefined = dynamic_cast<UserDefinedValueType const*>(type))
+		type = &userDefined->underlyingType();
```

Comment:
Now you have to do the same i had to do in the smtchecker ;)

Reply:
yep! But only because not all properties that matter are part of the interface.